### PR TITLE
add nosleep@0.5.0 w/ git auto-update

### LINKED ISSUE #9517: [Request] Add nosleep
**Library name:** nosleep.js
**Git repository url:** https://github.com/richtr/NoSleep.js
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dada0423**
Pull request for issue: #9517
Related issue(s): # #9414

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/richtr/NoSleep.js
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:473
   - Count of watchers:21
   - Count of forks:33
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9517, cc #9414, cc @richtr

### CODE REVIEW

File: ajax/libs/nosleep/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "nosleep",
+  "filename": "NoSleep.min.js",
+  "version": "0.5.0",
+  "description": "Prevent display sleep on all Android and iOS browsers",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/adasq/nosleep.git"
```
Comment (pvnr0082t) [inline ajax/libs/nosleep/package.json]: @dada0423 Wrong url.

File: ajax/libs/nosleep/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "author": "Rich Tibbett",
+  "license": "MIT",
+  "homepage": "https://github.com/richtr/NoSleep.js",
```
Comment (pvnr0082t) [inline ajax/libs/nosleep/package.json]: We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.

**Comment by pvnr0082t** [comment]
@dada0423 Any update?

File: ajax/libs/nosleep/package.json (hunk 1 above)
Comment (dada0423) [inline ajax/libs/nosleep/package.json]: fixed

File: ajax/libs/nosleep/package.json (hunk 2 above)
Comment (dada0423) [inline ajax/libs/nosleep/package.json]: I have removed homepage field. Please help me review it again. @pvnr0082t

**Comment by pvnr0082t** [comment]
LGTM
@cdnjs/intern2 Please help review this PR, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@kennynaoh then you should give this to me ... don't forget it ...

**Pull request merged**